### ISSUE TITLE
Builder broken - Undefined array key "type"

### ISSUE THREAD

**Issue by martin-ro** [laravel-filament/filament#7450]
### Package

filament/filament

### Package Version

v3.0.7

### Laravel Version

v10.17.1

### PHP Version

PHP 8.2

### Problem description

Simple Builder setup causes error `Undefined array key "type"` when saving/deleting.

Error: https://flareapp.io/share/87ne66yP

### Expected behavior

Adding and removing blocks without error.

### Steps to reproduce

1. Add to resource form:
```php
Forms\Components\Builder::make('builder')
                    ->blocks([
                        Forms\Components\Builder\Block::make('block')
                            ->schema([
                                Forms\Components\Toggle::make('enabled'),
                                Forms\Components\TextInput::make('title'),
                            ]),
                    ])
```

2. Add Block
3. Remove Block
4. Add Block
5. Remove Block --> Error `Undefined array key "type"`

### Reproduction repository

https://github.com/martin-ro/f3

### Relevant log output

```shell
https://flareapp.io/share/87ne66yP
```

**Comment by pboivin** [comment]
I'm also running into the same issue. I've been pulling my hair out silently trying to determine if the issue is on my end! I'll see if I can reproduce with your provided repository.

Edit: Yes, I could reproduce the issue!

**Comment by pboivin** [comment]
I'm digging into this a bit and I suspect it could be a bug in Livewire. Here's a breakdown of the `update` requests that happen in the Network tab of my dev tools when I'm following the steps described in the issue description:

![1](https://github.com/filamentphp/filament/assets/7805679/10da5bce-41fd-4782-a19e-649755f5c53d)

**Comment by martin-ro** [comment]
yes, I saw exactly the same thing.

**Comment by danharrin** [comment]
Just to let you know we are looking into this. I believe that #7421, #7450, #7475 and #7485 (and other duplicate issues) are all the same bug, related to Livewire's entangle feature.

**Comment by danharrin** [comment]
livewire/livewire#6289

**Comment by amondar-libs** [comment]
Still not worked on liveware 3.0.0-beta10. Same error: Undefined array key "type", but looks like fix already merged

**Comment by danharrin** [comment]
I believe livewire/livewire#6521 is a failing test for this

**Comment by mrahmadt** [comment]
I'm getting this error from just creating the example at https://filamentphp.com/docs/3.x/forms/fields/builder

any suggested workaround?

**Comment by danharrin** [comment]
Upgrade Livewire and Filament

**Comment by martin-ro** [comment]
I'm still experiencing this bug on v3.1.14

**Comment by RobertsG** [comment]
Also encountering this on with Filament v3.1.14 and livewire v3.2.6.

**Comment by danharrin** [comment]
Should be fixed in the next release. 5e58bea68018a02a904d9377a3a3baa0b08956a8

**Comment by dinandmentink** [comment]
Is there a fix/workaround for this in filament v2.0? I'm running the newest version.

**Comment by danharrin** [comment]
Feel free to submit a PR and I will merge/release it.
